install_helper.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+get_releases() {
+  curl --silent "https://api.github.com/repos/cdr/code-server/releases/latest" |
+    grep '"browser_download_url":\|"tag_name":'
+}
+
+linux_install() {
```

Comment:
Mac and linux install functions are pretty much duplicates. Let's have a single one that uses the values of bin_path and lib_path to install.

Reply:
The subtle differences make me think it'll be error-prone if we mix the logic. `.tgz` vs a `.zip`, the backup for mac, the release URL itself, etc.

Reply:
We can easily check for the extension and what command to use to extract.

I think we shouldn't worry about Mac. A home-brew formula is pretty simple and will be a much better option. Linux's 100 package systems is our only problem. Let's remove the Mac install.

Reply:
Agreed 👍